doc/dev/developer_guide/essentials.rst

```diff
@@ -304,6 +304,36 @@ The rest (including the actual backporting) will be taken care of by the
 .. _`tracker issue`: http://tracker.ceph.com/
 .. _`Stable Releases and Backports`: http://tracker.ceph.com/projects/ceph-releases/wiki
 
+Dependabot
+----------
+
+Dependabot is a github bot that scans the dependencies in the repositories for
+security vulnerabilities (CVEs). If a fix is available for a discovered CVE,
+Dependabot creates a pull request to update the dependency.
```

Comment:
I kinda want something like "the depended package" but can't come up with anything that doesn't sound stilted.

Reply:
"Dependent package" is somewhat common, but "dependency" alone is definitely more widespread.